compatLib/compatLibVU/src/main/java/app/lawnchair/compatlib/fourteen/ActivityOptionsCompatVU.java

```diff
@@ -13,9 +13,31 @@ public ActivityOptions makeCustomAnimation(
             Context context,
             int enterResId,
             int exitResId,
-            Runnable callback,
-            Handler callbackHandler) {
-        return null;
+            Handler callbackHandler,
+            Runnable startedListener,
+            Runnable finishedListener) {
+        return ActivityOptions.makeCustomAnimation(
```

Comment:
We should still invoke `makeCustomAnimation` here like?

https://cs.android.com/android/platform/superproject/main/+/main:packages/apps/Launcher3/quickstep/src/com/android/launcher3/uioverrides/QuickstepLauncher.java;l=1068